src/dali/manifest_builder.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2023 Neal Chambers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime, timezone
+from multiprocessing.pool import ThreadPool
+from typing import List, Set
+
+from dali.abstract_pair_converter_plugin import TransactionManifest
+from dali.abstract_transaction import AbstractTransaction
+from dali.in_transaction import InTransaction
+from dali.intra_transaction import IntraTransaction
+from dali.out_transaction import OutTransaction
+
+
```

Comment:
Can you add a comment describing what the manifest is, how it's used and what it contains?